code/modules/clothing/head/misc_special.dm

```diff
@@ -152,6 +158,7 @@
 	body_parts_covered = 0
 	siemens_coefficient = 1.5
 	item_icons = list()
+	style = STYLE_NEG_HIGH//you can't argue against this
```

Comment:
Yes I can, that's meme nerf that shouldn't be here.

Reply:
then argue with Kegdo.

Reply:
@Kegdo